internal/config.go

```diff
@@ -113,27 +130,28 @@ func MakeConfig() (*hazelcast.Config, error) {
 		return nil, fmt.Errorf("error reading Configuration at %s: %w", confPath, err)
 	}
 	if Token != "" {
-		config.Cluster.Cloud.Token = strings.TrimSpace(Token)
-		config.Cluster.Cloud.Enabled = true
+		config.Hazelcast.Cluster.Cloud.Token = strings.TrimSpace(Token)
+		config.Hazelcast.Cluster.Cloud.Enabled = true
+	}
+	if err := UpdateConfigWithSSL(&config.Hazelcast, &config.SSL); err != nil {
+		return nil, err
```

Comment:
Errors on MakeConfig are logged in the function since we do not have contextual information on "command" level about the errors.
To be in sync with rest of the flow, can you log the error here? Non-interactive mode does not print any error when this line is triggered

Reply:
Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/43/commits/56e501f45372988d99cf1bf7b0efa796cf365255